### DISCUSSION TITLE
Multiple map occurrences in config crashing app

### DISCUSSION THREAD

**Discussion by jwinkeler74** [microsoft/Dual-write-automations#16] [Q&A]
Good day.

Instead of setting the config file and group to keep trying a map when an error is occurred due to a reference (customers, product categories), I am attempting to add multiple occurrences of a map with different map versions (temporary version with reference fields removed). When I do this, the UI app crashes when trying to use the UI config editor.

Is this a scenario that is not supported in the app? Is the expectation in these cases (the same would hold for maps where the max lookup is exceeded when syncing CE --> F&SC) multiple config files would be created and run individually?

Thanks

**Comment by Hax1337** [comment]
Hey @jwinkeler74 apology for the late reply.

The multiple occurance of a map is not the intended design as you also just run one of the maps, means there should be only one occurance per unique map, same like its in the backend. 

Whats the use case to have it with multiple occurances?
